src/env.ts

```diff
@@ -11,7 +12,7 @@ const defaults: {[key: string]: string | null} = {
 };
 
 
-//********* Potentially trialscope specific code ********
+//* Potentially trialscope specific code *
```

Comment:
This code would be used if the API needs some sort of authorization. Not necessarily TrialScope specific